scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -338,6 +339,17 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
       })
     }
 
+    def currentIndentation: Int = {
+      val foundIndentation = countIndent(curr.pointPos)
+      if (foundIndentation < 0)
+        sepRegions.headOption match {
```

Comment:
instead of match, you could .fold(-1)(_.indent).

Reply:
Changed